renderer/components/Onboarding/Steps/Recover.js

```diff
@@ -40,32 +154,94 @@ class SeedWord extends React.Component {
+      index,
+      intl,
+      onPaste,
+      setFormApiFieldValue,
+      theme,
+      word,
+    } = this.props
+    const { value } = this.state
+
     return (
-      <Flex key={`word${index}`} alignItems="center" as="li" justifyContent="flex-start" my={1}>
-        <Label htmlFor={`word${index}`} mb={0} width={35}>
-          {index}
-        </Label>
-        <Input
-          field={`word${index}`}
-          hasMessage={false}
-          initialValue={word}
-          mask={this.mask}
-          onPaste={onPaste}
-          placeholder={intl.formatMessage({ ...messages.word_placeholder })}
-          validate={this.validate}
-          validateOnChange
-          variant="thin"
-          willAutoFocus={index === 1}
-        />
-      </Flex>
+      <Downshift
+        onSelect={selection => {
+          this.setState({
+            value: selection,
+          })
+          setFormApiFieldValue(selection)
+        }}
+      >
+        {({
+          getInputProps,
+          getItemProps,
+          getMenuProps,
+          getRootProps,
+          isOpen,
+          inputValue,
+          highlightedIndex,
+        }) => (
+          <Box sx={{ position: 'relative' }} {...getRootProps()}>
+            <Flex
+              key={`word${index}`}
+              alignItems="center"
+              as="li"
+              justifyContent="flex-start"
+              my={1}
+            >
+              <Label htmlFor={`word${index}`} mb={0} width={35}>
+                {index}
+              </Label>
+              <Input
+                {...getInputProps({
+                  field: `word${index}`,
+                  hasMessage: false,
+                  highlightOnValid: true,
+                  initialValue: word,
+                  mask: this.mask,
+                  onChange: this.onInputChange,
+                  onPaste,
+                  placeholder: intl.formatMessage({ ...messages.word_placeholder }),
+                  validate: this.validate,
+                  validateOnChange: true,
+                  value,
+                  variant: 'thin',
+                  willAutoFocus: index === 1,
+                  fieldState: {
+                    asyncError: false,
+                    error: value.length ? this.validate(value) : formState.errors[`word${index}`],
+                    touched: formState.touched[`word${index}`],
+                    value,
+                  },
+                })}
+              />
+            </Flex>
+            {isOpen && inputValue && (
+              <SeedWordList
+                {...{
+                  getMenuProps,
+                  highlightedIndex,
+                  getItemProps,
+                  inputValue,
+                  bip39WordMap,
+                  theme,
+                }}
+              />
+            )}
+          </Box>
+        )}
+      </Downshift>
     )
   }
 }
 
 const SeedWordWithIntl = compose(
   withFormApi,
-  injectIntl
+  injectIntl,
+  withTheme
```

Comment:
This is probably used just for the bg here 
https://github.com/LN-Zap/zap-desktop/pull/2896/files#diff-378f70cc56cf8c3617c6784bc0c1dbb1R70
and should go once you refactor that

Reply:
Yeah, will do.

Reply:
Done.